src/Sulu/Component/Content/Types/Rlp/Strategy/TreeStrategy.php

```diff
@@ -42,6 +42,13 @@ public function __construct(
         );
     }
 
+    public function getChildPart($resourceSegment)
+    {
+        $resourceSegmentParts = explode('/', $resourceSegment);
```

Comment:
Well, I think in this case it doesn't matter if the path is absolute, and this code seems also to work if it's not absolute:

```php
php > var_dump(explode('/', 'asdf'));
array(1) {
  [0] =>
  string(4) "asdf"
}
```

Why should we make it more complicated than necessary?